components/BottomAppBar/examples/supplemental/BottomAppBarTypicalUseSupplemental.m

```diff
@@ -130,7 +139,7 @@ - (UITableViewCell *)tableView:(UITableView *)tableView
   cell.layoutMargins = UIEdgeInsetsZero;
   cell.textLabel.text = self.listItems[indexPath.item];
   [self.fabVisibilitySwitch removeFromSuperview];
-  if (indexPath.row == (NSInteger)(self.listItems.count - 1)) {
+  if ([self.listItems[indexPath.row] isEqualToString:@"Visible FAB"]) {
```

Comment:
This is probably a good change, but it's unrelated to this PR. Can you break this out to a separate pull request?

Reply:
Since the implementation was changed to using contentInset, I think the original implementation here is good. I reverted this one in the code review change commit.